### DISCUSSION TITLE
sftp_to_s3 operator

### DISCUSSION THREAD

**Discussion by madhusudhan0542** [apache/airflow#16970] [Q&A] [answered]
I was able to transfer a single file using the sftp_to_s3 operator from sftp to s3
but, when I am attempting to transfer multiple files by give the path of directory/folder or like *.txt in sftp_path parameter it says no such file found

I had tried different ways and also referred to various sources could not find any proper solution 
could you please help me out to transfer multiple files using the sftp_to_s3 operator 

Thank you in advance

**Comment by potiuk** [comment, marked as answer]
I think you need to write your custom operator if you want to have this functionality - it seems it's not there. It would be great to contribute it back as well so that others can use it).

**Reply by madhusudhan0542** [reply]
> I think you need to write your custom operator if you want to have this functionality - it seems it's not there. It would be great to contribute it back as well so that others can use it).

ok Thank You

**Reply by Sandey243** [reply]
Hi, 
Have been able to write a custom operator for your requirement, I am in need of same operator, if you have written it already, please can you share it with me
